charts/chart-colors.ts

```diff
@@ -0,0 +1,334 @@
+      "#bf9727",
+      "#607100",
+      "#00734c",
+      "#704489",
+      "#01acca",
+      "#024e76",
+      "#f09100",
+      "#ea311f",
+      "#c6004b",
+      "#7570b3",
+      "#666666",
+      "#333333",
+    ],
+  },
+  {
+    name: "vibrant-rainbow",
+    colors: [
+      "#fffb00",
+      "#f5cb11",
+      "#9fd40c",
+      "#46e39c",
+      "#32b8a6",
+      "#7ff2fa",
+      "#ad00f2",
+      "#c461eb",
+      "#eb7200",
+      "#e8a784",
+      "#bf2e2e",
+      "#6c7000",
+    ],
+  },
+  {
+    name: "ocean-bay",
+    colors: [
+      "#191921",
+      "#11495c",
+      "#78b1c2",
+      "#454f4b",
+      "#8f8f82",
+      "#9be0c0",
+      "#87b051",
+      "#f7ec88",
+      "#ebdcc1",
+      "#dbb658",
+      "#c43541",
+      "#75351e",
+    ],
+  },
+  {
+    name: "prairie-summer",
+    colors: [
+      "#332424",
+      "#751555",
+      "#d47013",
+      "#d68989",
+      "#211173",
+      "#82aad6",
+      "#7bfaeb",
+      "#6ec9a8",
+      "#6b6408",
+      "#eada40",
+      "#ccc54a",
+      "#1fc235",
+    ],
+  },
+  {
+    name: "pastel-chalk",
+    colors: [
+      "#fffd99",
+      "#f5e6a4",
+      "#c1d48c",
+      "#b8e3d0",
+      "#a0b8b5",
+      "#cbf7fa",
+      "#d791f2",
+      "#dfc1eb",
+      "#f2b983",
+      "#e8c4b2",
+      "#bf8e8e",
+      "#94995c",
+    ],
+  },
+  {
+    name: "seq-yellow-orange-red-bright",
+    colors: [
+      "#910000",
+      "#b1260b",
+      "#c0370f",
+      "#e05919",
+      "#ef6a1d",
+      "#ff7b22",
+      "#ffa143",
+      "#ffb454",
+      "#ffda74",
+      "#ffed85",
+    ],
+  },
+  {
+    name: "seq-reds-bright",
+    colors: [
+      "#57453b",
+      "#7b4238",
+      "#9f4036",
+      "#c23d33",
+      "#d7483c",
+      "#ec5244",
+      "#f3696c",
+      "#f9816c",
+      "#ffc4ae",
+      "#fff0dc",
+    ],
+  },
+  {
+    name: "seq-purples-bright",
+    colors: [
+      "#4e465c",
+      "#5a4a78",
+      "#695291",
+      "#775baa",
+      "#8663c3",
+      "#946bdc",
+      "#aa89e8",
+      "#c1a6f3",
+      "#d7c4ff",
+      "#e6e1ff",
+    ],
+  },
+  {
+    name: "seq-blues-bright",
+    colors: [
+      "#404d54",
+      "#435c6c",
+      "#48799d",
+      "#4b88b6",
+      "#4d96ce",
+      "#50a5e7",
+      "#74bbed",
+      "#98d0f3",
+      "#bce6f9",
+      "#e6faff",
+    ],
+  },
+  {
+    name: "seq-greens-bright",
+    colors: [
+      "#39544c",
+      "#386757",
+      "#368165",
+      "#359b73",
+      "#33b581",
+      "#4bc392",
+      "#64d2a2",
+      "#7ce0b3",
+      "#cbf6d9",
+      "#f4ffea",
+    ],
+  },
+  {
+    name: "seq-browns-bright",
+    colors: [
+      "#524834",
+      "#715b38",
+      "#8f6e3c",
+      "#ae8140",
+      "#cc9444",
+      "#eba748",
+      "#eeb664",
+      "#f0c47f",
+      "#f9e0b7",
+      "#fff8eb",
+    ],
+  },
+];
+
+export const ChartColors: { palette: string[]; sets: ColorSet[] } = {
+  palette: ColorPalette,
+  sets: ColorSets,
+};
+
+export function colorFromColorSet(
```

Comment:
Not sure I understand the need for this helper. Wouldn't it be simpler to access the color directly?

```ts
colorFromColorSet(5, someColorSet);

// vs

someColorSet.colors[5]
```

Reply:
Yeah me either. I'll remove. It was from dashboard code.